tests/source/test_ini.py

```diff
@@ -0,0 +1,12 @@
+from dffml.source.ini import INISource, INISourceConfig
+from dffml.util.testing.source import FileSourceTest
+from dffml.util.asynctestcase import AsyncTestCase
+
+
+class TestINISource(FileSourceTest, AsyncTestCase):
+    async def setUpSource(self):
```

Comment:
Don't involve `FileSourceTest`, just write a couple little tests with comments that test the basic functionality